content/en/news/releases/1.21.x/announcing-1.21/change-notes/index.md

```diff
@@ -0,0 +1,320 @@
+
+- **Added** support for deleting multiple waypoints at once via `istioctl x waypoint delete <waypoint1> <waypoint2> ...`
+
+- **Added** a flag `--all` to `istioctl experimental waypoint delete` to delete all Waypoint resources in a given namespace.
+
+- **Added** an analyzer to warn users if they set the `selector` field instead of the `targetRef` field for specific Istio resources, which will cause the resource to be ineffective.
+  ([Issue #48273](https://github.com/istio/istio/issues/48273))
+
+- **Added** message IST0167 to warn users that policies, such as Sidecar, will have no impact when applied to ambient namespaces.
+  ([Issue #48105](https://github.com/istio/istio/issues/48105))
+
+- **Added** bootstrap summary to all config dumps' summary.
+
+- **Added** completion for Kubernetes pods for some commands that can select pods, such as `istioctl proxy-status <pod>`.
+
+- **Added** `--wait` option to `istioctl experimental waypoint apply` command.  ([Issue #46297](https://github.com/istio/istio/issues/46297))
+
+- **Added** `path_separated_prefix` to the MATCH column in the output of `pc routes` command.
+
+- **Fixed** an issue where sometimes control plane revisions and proxy versions were not obtained in the bug report.
+
+- **Fixed** an issue where `istioctl tag list` command didn't accept `--output` flag.
+  ([Issue #47696](https://github.com/istio/istio/issues/47696))
+
+- **Fixed** an issue where the default namespace of envoy and proxy dashboard command was not set to the actual default namespace.
+
+- **Fixed** an issue where the IST0158 message was incorrectly reported when the `imageType` field was set to `distroless` in mesh config.
+  ([Issue #47964](https://github.com/istio/istio/issues/47964))
+
+- **Fixed** an issue where `istioctl experimental version` has no proxy info shown.
+
+- **Fixed** an issue where the IST0158 message was incorrectly reported when the `imageType` field was set by the `ProxyConfig` resource, or the resource annotation `proxy.istio.io/config`.
+
+- **Fixed** an issue `proxyconfig ecds` didn't show all `EcdsConfigDump`.
+
+- **Fixed** injector list has duplicated namespace shown for the same injector hook.
+
+- **Fixed** `istioctl analyze` didn't work correctly when analyzing files containing resources that already exist in the cluster.
+  ([Issue #44844](https://github.com/istio/istio/issues/44844))
+
+- **Fixed** an issue where `istioctl analyze` reported errors for empty file resource blocks.
+  ([Issue #45653](https://github.com/istio/istio/issues/45653))
+
+- **Fixed** an issue where the External Control Plane Analyzer may not work in some remote control plane setups.
+
+- **Removed** `--rps-limit` flag for `istioctl bug-report` and **added** `--rq-concurrency` flag.
+The bug reporter will now limit request concurrency instead of limiting request rate to the kube
+API.
```

Comment:
```suggestion
- **Removed** the `--rps-limit` flag for `istioctl bug-report` and **added** the `--rq-concurrency` flag.
The bug reporter will now limit request concurrency instead of limiting request rate to the kube
API.
```